resources/tests/fcnet-setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+main() {
+    devs=$(ls /sys/class/net | grep -v lo)
+    for dev in $devs; do
+        mac_ip=$(ip link show dev $dev \
+            | grep link/ether \
+            | grep -Po "(?<=06:00:)([0-9a-f]{2}:?){4}"
+        )
+        ip=$(printf "%d.%d.%d.%d" $(echo "0x${mac_ip}" | sed "s/:/ 0x/g"))
+        ip addr add "$ip/30" dev $dev
```

Comment:
Maybe a more robust solution here would be to have this script generate the required entries in ` /etc/network/interfaces` and have the script run in a service that has Before=network-pre.target set.

This way, we would leave the network device lifetime handling to NetworkManager or similar services that manage the network stack.
This helps if maybe sometime we want to run tests that change the state of the network devices and makes the configuration more standard and easy to manage.